src/lib.rs

```diff
@@ -3334,6 +3334,21 @@ impl Build {
                     // Use the GNU-variant to match other Unix systems.
                     name = format!("g{}", tool).into();
                     self.cmd(&name)
+                } else if target.os == "vxworks" {
+                    name = format!("wr-{}", tool).into();
+                    self.cmd(&name)
+                } else if target.os == "nto" {
+                    name = match target.arch {
+                        "i586" | "x86" => format!("ntox86-{}", tool).into(),
+                        "aarch64" | "x86_64" => format!("nto{}-{}", target.arch, tool).into(),
```

Comment:
Nit: `i586` is never a `target_arch` that `rustc` will produce.

```suggestion
                        "x86" | "aarch64" | "x86_64" => format!("nto{}-{}", target.arch, tool).into(),
```